workflows/proteomics/clinicalmp/clinicalmp-verification/iwc_clinicalmp_verification_workflow.ga

```diff
@@ -0,0 +1,933 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "WF3- Peptide verification/validaiton workflow",
+    "comments": [],
+    "creator": [
+        {
+            "class": "Organization",
+            "identifier": "0000-0001-9818-0537",
+            "name": "Subina Mehta"
+        }
+    ],
+    "format-version": "0.1",
+    "license": "CC-BY-4.0",
+    "name": "iwc_clinicalmp_verification_workflow",
+    "report": {
+        "markdown": "\n# Workflow Execution Report\n\n## Workflow Inputs\n```galaxy\ninvocation_inputs()\n```\n\n## Workflow Outputs\n```galaxy\ninvocation_outputs()\n```\n\n## Workflow\n```galaxy\nworkflow_display()\n```\n"
+    },
+    "steps": {
+        "0": {
+            "annotation": "",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/galaxyp/dbbuilder/dbbuilder/0.3.4",
+            "errors": null,
+            "id": 0,
+            "input_connections": {},
+            "inputs": [],
+            "label": "Human_UniProt_Isoforms_FASTA",
```

Comment:
All of the labels are targeted at users, there's no need for underscores or file extensions in these labels

Reply:
Made changes.